website/content/docs/platform/k8s/helm/run.mdx

```diff
@@ -466,6 +475,10 @@ to be unsealed:
 $ kubectl exec -ti <name of pod> -- vault operator unseal
 ```
 
+(If using Autpilot Automated Upgrades and the cluster size is at least 3,
```

Comment:
```suggestion
(If using Autopilot Automated Upgrades and the cluster size is at least 3,
```

Reply:
In this section, are we guiding people using RollingUpdate and autopilot upgrades?  I'm confused because why are we asking people to delete pods above prior to the new pods being ready if they're using autopilot upgrades?

Reply:
Also how does cluster size come into this?